localization/replace/english/wc_title_replacements_l_english.yml

```diff
@@ -229,4 +229,11 @@
  b_ramu:0 "Ra'mu"
  b_ronda:0 "Ron'da"
  b_larne:0 "Lar'ne"
- b_carrion:0 "Carrion"
\ No newline at end of file
+ b_carrion:0 "Carrion"
+ b_keeshan:0 "Ilgalar"
+ b_keeshan_adj:0 "Ilgalarian"
```

Comment:
There is already a barony named Ilgalar, and it's in the same county as Keeshan.

Reply:
fixed